c2rust-analyze/src/c_void_casts.rs

```diff
@@ -303,7 +388,7 @@ impl<'tcx> CVoidCasts<'tcx> {
     ///
     /// [`*c_void`]: core::ffi::c_void
     fn insert_all_from_body(&mut self, body: &Body<'tcx>, tcx: TyCtxt<'tcx>) {
-        for bb_data in body.basic_blocks().iter() {
+        for (block, bb_data) in body.basic_blocks().iter().enumerate() {
```

Comment:
```suggestion
        for (block, bb_data) in body.basic_blocks().iter_enumerated() {
```

Then you don't need the `BasicBlock::from_usize(block)`s.

Reply:
89fc7fd9